pfcpiface/telemetry.go

```diff
@@ -4,12 +4,52 @@
 package main
 
 import (
+	"log"
 	"net/http"
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
+var (
+	pfcpEstReq = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "pfcp_est_req_processed_total",
+		Help: "The total number of processed Pfcp Establishment Requests",
+	})
+	pfcpModReq = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "pfcp_mod_req_processed_total",
+		Help: "The total number of processed Pfcp Modification Requests",
+	})
+	pfcpDelReq = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "pfcp_del_req_processed_total",
+		Help: "The total number of processed Pfcp Deletion Requests",
+	})
+	pfcpAssocReq = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "pfcp_assoc_req_processed_total",
+		Help: "The total number of processed Pfcp Association setup Requests",
+	})
+	pfcpEstRsp = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "pfcp_est_rsp_sent_total",
+		Help: "The total number of Pfcp Establishment Responses sent",
+	}, []string{"result"})
+	pfcpModRsp = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "pfcp_mod_rsp_sent_total",
+		Help: "The total number of Pfcp Modification Responses sent",
+	}, []string{"result"})
+	pfcpDelRsp = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "pfcp_del_rsp_sent_total",
+		Help: "The total number of Pfcp Deletion Responses sent",
+	}, []string{"result"})
+	pfcpAssocRsp = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "pfcp_assoc_rsp_sent_total",
+		Help: "The total number of Pfcp Association setup Responses sent",
+	}, []string{"result"})
```

Comment:
replace all these with a single CounterVec called PFCP messages and label with message type and direction instead
https://pkg.go.dev/github.com/prometheus/client_golang@v1.9.0/prometheus#CounterVec
https://pkg.go.dev/github.com/prometheus/client_golang@v1.9.0/prometheus#example-CounterVec

Reply:
Success/Failure labels can also be added

Reply:
done